ouroboros-network/src/Ouroboros/Network/Protocol/LocalStateQuery/Client.hs

```diff
@@ -93,6 +98,62 @@ data ClientStQuerying block query m a result = ClientStQuerying {
       recvMsgResult :: result -> m (ClientStAcquired block query m a)
     }
 
+-- | Transform a 'LocalStateQueryClient' by mapping over the tx header and the
+-- chain tip values.
+--
+-- Note the mapping is contravariant in both types since they are outputs of
+-- the protocol, not inputs.
```

Comment:
The comment is not right, it is covariant in block and query types and contravariant in the result type.

Reply:
Thanks. Yes.